appservice/src/createAppService/SiteCreateStep.ts

```diff
@@ -29,11 +29,11 @@ export class SiteCreateStep extends AzureWizardExecuteStep<IAppServiceWizardCont
     public async execute(wizardContext: IAppServiceWizardContext): Promise<IAppServiceWizardContext> {
         if (!wizardContext.site) {
             const creatingNewApp: string = localize('CreatingNewApp', 'Creating {0} "{1}"...', getAppKindDisplayName(wizardContext.newSiteKind), wizardContext.newSiteName);
-            await window.withProgress({ location: ProgressLocation.Notification, title: creatingNewApp }, async (): Promise<void> => {
-                ext.outputChannel.appendLine(creatingNewApp);
-                const client: WebSiteManagementClient = new WebSiteManagementClient(wizardContext.credentials, wizardContext.subscriptionId, wizardContext.environment.resourceManagerEndpointUrl);
-                addExtensionUserAgent(client);
-                wizardContext.site = await client.webApps.createOrUpdate(nonNullValueAndProp(wizardContext.resourceGroup, 'name'), nonNullProp(wizardContext, 'newSiteName'), {
+            if (wizardContext.progress) { wizardContext.progress.report({message: creatingNewApp}); }
```

Comment:
Did you make sure to try out the functions scenario? I feel like there's some function-specific steps that you didn't change (storage account?)

Also make sure to try 'appService.advancedCreation' if you didn't already.